years_to_hours.rb

```diff
@@ -0,0 +1,21 @@
+puts "Enter a number of years"
+years = gets.chomp
+years = years.to_i
+hours = years * 365 * 24
+puts "That's #{hours} hours."
+sleep 1
+
+
+puts "Enter a number of decades"
+decades = gets.chomp
+decades = decades.to_i
+minutes = years * 365 * 24 * 60
```

Comment:
Looks like you got a calculation mistake. We need to use `decade` here instead of `years`.

Reply:
I have created a new pull request that has corrected the error:
https://github.com/Hannahlyzer/Backend-sqlruby/pull/2